audio/portaudio/Portfile

```diff
@@ -3,8 +3,7 @@
 PortSystem          1.0
 
 name                portaudio
-version             19.20140130
-revision            2
+version             190600.20161030
 categories          audio devel
```

Comment:
This revision number seems wrong. I recognize it is what the distribution file is named, but going from version 19. to version 190600. seems rather unexpected.

Reply:
I thought they'd changed their versioning scheme but it seems you're right. It's actually supposed to be 19.6, and I have no idea why they wrote it out like that.

Reply:
Either way... it's ugly but I'd rather leave it like this if the tradeoff is not having to check and possibly modify a regex search whenever they decide to release a new version.

Reply:
I think the odds of a new version are low, but I do in fact think you need to call this 19.20161030 and do some ugly manual fixup.

Reply:
In that case, the more correct number would be 19.6.20161030.
As for new versions... while they (evidently) don't make stable releases often, if you go to their git, you'll notice that the project is still, in fact, active.

Reply:
I'm thinking we could have 
```
version    19.6
epoch    20161030
```

And construct the proper distname from that , something like replacing ```\.(\d)$``` to convert .6 to 0600, but I cannot figure out the proper syntax, if you wouldn't mind helping me with this?

Reply:
That's not what the epoch field is for. Or rather, we'd have "epoch 1", not "epoch 20161030", and I'd suggest not doing this but just continuing with the last format for the version number.

Reply:
They seem to call this version 19.6.0. See https://app.assembla.com/wiki/show/portaudio/pa_stable_v190600_20161030

Reply:
Yeah I know that's not really the purpose of epoch (it even says so in the documentation) but I'm thinking of having to reconcile keeping the version format with fetching the proper source, without having to hardcode it.

Reply:
It isn't merely not the purpose of the epoch. The epoch number is a small integer, not a large integer. You don't call it 20161030 regardless, since there isn't one now, it would be 1.

And you're going to have to hardcode stuff. That's not such a problem considering it isn't even clear that the developers intend to ever release again. If they do, it's clearly not going to happen more than once every couple of years.